tests/installation/src/constants.ts

```diff
@@ -30,6 +30,10 @@ export const SANITY_TEST_REPORTS_DIR: string = path.resolve(SANITY_TEST_ROOT_DIR
 
 // 45 minutes timeout for install conv. build
 export const TEST_TIMEOUT_CONVENIENCE_BUILD: number = 60 * 60 * 1000;
+
+// 30 minutes timeout for install docker build
+export const TEST_TIMEOUT_DOCKER_BUILD: number = 30 * 60 * 1000;
```

Comment:
I think this value is not used any more? Or do you plan to use it in `tests/installation/src/__tests__/basic/install-docker.ts`?

Reply:
no plans, i can remove it